group_vars/dev.yml

```diff
@@ -0,0 +1,101 @@
+---
+copr_instance: dev
+copr_instance_tag: devel
+copr_zone: us-east-1c
+
+backend:
+  name: copr-backend
+  old_instance_id: i-0ba958e718cb91fa3
+  new_instance_id: i-0007e54a85c0e56b1
```

Comment:
0007e54a85c0e56b1 is id of current copr-backend-dev. And  0ba958e718cb91fa3 does not exists. Did you already migrated the stage?

Reply:
Yes, I already migrated it...